lifecycle-operator/controllers/common/helperfunctions.go

```diff
@@ -41,35 +40,7 @@ func GetOldStatus(name string, statuses []klcv1alpha3.ItemStatus) apicommon.Kept
 	return oldstatus
 }
 
-func setEventMessage(phase apicommon.KeptnPhaseType, reconcileObject client.Object, longReason string, version string) string {
-	if version == "" {
-		return fmt.Sprintf("%s: %s / Namespace: %s, Name: %s", phase.LongName, longReason, reconcileObject.GetNamespace(), reconcileObject.GetName())
-	}
-	return fmt.Sprintf("%s: %s / Namespace: %s, Name: %s, Version: %s", phase.LongName, longReason, reconcileObject.GetNamespace(), reconcileObject.GetName(), version)
-}
-
-func setAnnotations(reconcileObject client.Object, phase apicommon.KeptnPhaseType) map[string]string {
-	if reconcileObject == nil || reconcileObject.GetName() == "" || reconcileObject.GetNamespace() == "" {
-		return nil
-	}
-	annotations := map[string]string{
-		"namespace": reconcileObject.GetNamespace(),
-		"name":      reconcileObject.GetName(),
-		"phase":     phase.ShortName,
-	}
-
-	piWrapper, err := interfaces.NewEventObjectWrapperFromClientObject(reconcileObject)
-	if err == nil {
-		copyMap(annotations, piWrapper.GetEventAnnotations())
-	}
-
-	annotationsObject := reconcileObject.GetAnnotations()
-	annotations["traceparent"] = annotationsObject["traceparent"]
-
-	return annotations
-}
-
-func copyMap[M1 ~map[K]V, M2 ~map[K]V, K comparable, V any](dst M1, src M2) {
+func CopyMap[M1 ~map[K]V, M2 ~map[K]V, K comparable, V any](dst M1, src M2) {
```

Comment:
we should bump to Go 1.21 so we can get rid of this function :)

Reply:
ok, but this should be a follow-up ticket and this part will be covered as part of that

Reply:
https://github.com/keptn/lifecycle-toolkit/issues/2421

Reply:
yeah absolutely not part of this PR :)